### PR TITLE
repos: 4.16: Use RHEL 9.4 EUS repos for RHCOS

### PR DESCRIPTION
**Pull request by travier**
repos: 4.16: Use RHEL 9.4 EUS repos for RHCOS

---

repos: 4.16: Remove now unused RHEL 9.2 repos

### CODE REVIEW

**Review by jbtrystram — APPROVED** [review]
/approved

**Comment by travier** [comment]
CC @jupierce @sosiouxme @bradmwilliams @AlexNPavel for approval

File: core-services/release-controller/_repos/ocp-4.16-rhel94.repo [hunk 1]
```diff
@@ -36,7 +36,7 @@ failovermethod = priority
 
 [rhel-9.4-nfv]
 name = rhel-9.4-nfv
-baseurl = https://cdn.redhat.com/content/beta/rhel9/9/x86_64/nfv/os
+baseurl = https://cdn.redhat.com/content/e4s/rhel9/9.4/x86_64/nfv/os/
```
Comment (c4rt0) [inline core-services/release-controller/_repos/ocp-4.16-rhel94.repo:39]: Not sure what `e4s` is, but it differs from the `eus` above... hence this comment.

Comment (travier) [reply core-services/release-controller/_repos/ocp-4.16-rhel94.repo:39]: The NFV repos are in a different place as they have a different lifecycle. See for RHEL 9.2: https://github.com/openshift/release/blob/master/core-services/release-controller/_repos/ocp-4.15-rhel92.repo#L39

Comment (travier) [reply core-services/release-controller/_repos/ocp-4.16-rhel94.repo:39]: There are some hints in https://access.redhat.com/articles/3751271

Comment (c4rt0) [reply core-services/release-controller/_repos/ocp-4.16-rhel94.repo:39]: That explains it, thanks.
/lgtm

**Review by danilo-gemoli — APPROVED** [review]

**Pull request merged**